pkg/api/versionutils.go

```diff
@@ -34,10 +34,10 @@ func (r VersionRange[V]) String() string {
 // If either range is invalid, or no such version exists (i.e. the ranges are disjoint), then the
 // returned values will be (0, false).
 func (r VersionRange[V]) LatestSharedVersion(cmp VersionRange[V]) (_ V, ok bool) {
-	max := util.Min(r.Max, cmp.Max)
```

Comment:
Now that #1035 has been merged, we can also get rid of `util.{Min,Max}` -- I'll open a follow up :)

Reply:
Thanks!